### PR TITLE
added known issue for topology argo cd pull model

### PR DESCRIPTION
**Pull request by jc-berger**
See Jira: https://issues.redhat.com/browse/ACM-6954

### CODE REVIEW

File: release_notes/known_issues_application.adoc [hunk 1]
```diff
@@ -29,6 +29,12 @@ For more about deprecations and removals, see xref:../release_notes/deprecate_re
 
 See the following known issues for the application lifecycle component.
 
+[#argo-pull-model-topology]
+== Topology does not correctly display for Argo CD pull model `ApplicationSet` application 
+//2.9.0: 3910
```
Comment (jc-berger) [inline release_notes/known_issues_application.adoc:34]: At Doc Team reviewers, 

Did I document this known issue change correctly for 2.9? 

All the examples I had in this doc were for 2.8, so I want to ensure I've documented it correctly so they doc changes render properly for 2.9. Thanks!

**Review by fxiang1 — APPROVED** [review]
LGTM! Thanks Jacob!

File: release_notes/known_issues_application.adoc [hunk 2]
```diff
@@ -29,6 +29,12 @@ For more about deprecations and removals, see xref:../release_notes/deprecate_re
...
+
+When you use the Argo CD pull model to deploy `ApplicationSet` applications, the application resource names are being customized, and they might appear different for each cluster. When this happens, the topology does not display your application correctly.
```
Comment (dockerymick) [inline release_notes/known_issues_application.adoc]: It seems like the resources are actively being customized during the deployment in the original statement. I imagine that the deployment is created or "customized first and then deployed. I suggest the following change:

```suggestion
When you use the Argo CD pull model to deploy `ApplicationSet` applications and the application resource names are customized, the resource names might appear different for each cluster. When this happens, the topology does not display your application correctly.
```

File: release_notes/known_issues_application.adoc (hunk 1 above)
Comment (dockerymick) [reply release_notes/known_issues_application.adoc:34]: You did doc this correctly! I would remove the extra space in the hidden comment, but that's really not important lol the rest of your updates look good!

**Review by dockerymick — CHANGES_REQUESTED** [review]

**Review by dockerymick — APPROVED** [review]
Thanks for accepting my suggestions! The changes look good to me

**Pull request merged**